src/docs/sdk/hub_and_scope_refactoring.mdx

```diff
@@ -25,14 +25,57 @@ This is necessary because the Sentry SDKs should be able to capture all spans fr
 
 We have different concepts like a `Hub` and `Scope`s in the Unified API and those concepts sometimes leak to the end user of our SDKs. Those concepts are complicated and the users should never need to think about what a `Hub` or a `Scope` is and if they should clone them or not. 
 
-
 ## What Is the Outcome of the Refactoring?
 
 1. We want to make Sentry SDKs 100% compatible with OTel performance data, so users can choose to use OTel tooling for their performance monitoring and still see their data in Sentry. (And Sentry error events are still connected to the spans captured by OTel)
 1. We want to make it easier for the users to do custom performance instrumentation with as little as possible mental overhead. The `Hub` will be gone.
 
+<Alert title="Note" level="info">
+The decision of removing the Hub from all Sentry SDKs was confirmed in a meeting on 2024-05-03 with most of the SDK engineers and Armin present. This decision is final.
+</Alert>
+
 This is why we started this journey with an <Link to="https://github.com/getsentry/rfcs/pull/122">RFC</Link>.
 
+## Why do we want to get rid of hubs?
+
+Hubs and scopes is a complicated system, that can easily be misused (also see [here](https://develop.sentry.dev/sdk/research/performance/#scope-propagation)). It requires us to create new hubs which in turn breaks breadcrumbs and other features. For the future we want to have one consistent system, that forks off in the same way as OTEL `Context` does.
+
+Regardless of switching our performance over to OTel, we should merge hubs and scopes. So hubs should be removed from all Sentry SDKs.
+
+### Copy-on-write
+
+Getting rid of hubs also implies a soft copy-on-write.
+
+In our current system with hubs and scopes, `Scope` is always mutable and what makes it somewhat immutable is pushing/popping scopes and creating new hubs. This however has to be done manually.
+
+If an async operation is started halfway in between handling a request, a new hub clone has to be created to achieve copy-on-write. In case we don't create a new hub clone, popping a `Scope` also affects the original execution, leading to lost data or worst case exceptions.
+
+After merging hubs and scopes, scopes can be forked at any time and a user no longer has to take care of it, as it should automatically do the right thing. With this new system, the async execution can manipulate scope without affecting other execution.
+
+This soft copy-on-write relies on `Scope` being forked in certain cases:
```

Comment:
Somewhere else in this doc it says:

> whenever a context is mutated in OTel a new context is created

I'm not sure if we can achieve this in all SDKs. I'm also not sure if we need to. Most likely intercepting `Context` storage is good enough. And only forking when a new OTEL span is created should also suffice. I was not able to confirm these statements yet.